src/main/kotlin/com/dsrm/dsrmbackend/validators/UserRolesConstraint.kt

```diff
@@ -0,0 +1,16 @@
+package com.dsrm.dsrmbackend.validators
+
+import javax.validation.Constraint
+import javax.validation.Payload
+import kotlin.reflect.KClass
+
+
+@MustBeDocumented
+@Constraint(validatedBy = [UserRolesValidator::class])
+@Target(AnnotationTarget.FIELD)
+@Retention(AnnotationRetention.RUNTIME)
+annotation class UserRolesConstraint(
+    val message: String = "roles.not.exist",
```

Comment:
We want to use string interpolation. It should be {roles.not.exist.message} and put in ValidationMessage.properties